src/funit/fhamcrest/BaseDescription.F90

```diff
@@ -305,6 +305,7 @@ function description_of_real64(value) result(string)
     end function description_of_real64
 
 
+#if (defined(_ISO_REAL128) && (_ISO_REAL64 != _REAL_DEFAULT_KIND) && (_ISO_REAL128 != _DOUBLE_DEFAULT_KIND))
```

Comment:
Should this (and line 337) be `_ISO_REAL_128` everywhere like line 118 above?

Reply:
yes